.github/workflows/test.yml

```diff
@@ -21,7 +21,9 @@ jobs:
           - centos-7-x64
           - centos-8-x64
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
+      with:
+        ref: master
```

Comment:
I think the warning was about this `@master` specifically, `with.ref: master` is something else isn't it?

Reply:
Totally misunderstood the problem there, fixed.